test/controllers/actions/AuthActionSpec.scala

```diff
@@ -102,6 +107,43 @@ class AuthActionSpec extends SpecBase {
       }
     }
 
+    "the user has an HMRC-MDR-ORG enrolment and internalId" - {
```

Comment:
@BriWak Can we add a few more test cases, 

- when there is no  MDRID  [Seq(EnrolmentIdentifier("MDRID", "")),]
- when there are no identifiers
- When there is no key "HMRC-MDR-ORG"

Reply:
They should all be added now